apps/web/src/app/(admin)/backoffice/templates/page.tsx

```diff
@@ -0,0 +1,196 @@
+import useEvaluationTemplates from '$/stores/evaluationTemplates'
+
+import { useEvaluationConfiguration } from '../../../(private)/evaluations/_components/CreateEvaluationModal'
+
+export default function AdminPage() {
+  const { data: evaluationTemplates } = useEvaluationTemplates()
+
+  return (
+    <div className='w-full max-w-[1250px] m-auto px-4 py-8 pt-0 flex flex-col gap-8'>
+      <NewEvaluationTemplate />
+      <EvaluationTemplatesTable evaluationTemplates={evaluationTemplates} />
+    </div>
+  )
+}
+
+function EvaluationTemplatesTable({
+  evaluationTemplates,
+}: {
+  evaluationTemplates: EvaluationTemplateWithCategory[]
+}) {
+  const { destroy } = useEvaluationTemplates()
+  if (!evaluationTemplates.length) return null
+
+  return (
+    <TableWithHeader
+      title='Evaluation Templates'
+      table={
+        <Table>
+          <TableHeader>
+            <TableRow>
+              <TableHead>Name</TableHead>
+              <TableHead>Category</TableHead>
+              <TableHead>Description</TableHead>
+              <TableHead>Configuration</TableHead>
+              <TableHead />
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {evaluationTemplates.map((template) => (
+              <TableRow
+                key={template.id}
+                className='cursor-pointer border-b-[0.5px] h-12 max-h-12 border-border'
+              >
+                <TableCell>
+                  <Text.H4 noWrap>{template.name}</Text.H4>
+                </TableCell>
+                <TableCell>
+                  <Text.H4 noWrap>{template.category}</Text.H4>
+                </TableCell>
+                <TableCell>
+                  <div className='flex flex-row gap-1 justify-between items-center'>
+                    <div className='flex-auto'>
+                      <Text.H4>{template.description}</Text.H4>
+                    </div>
+                  </div>
+                </TableCell>
+                <TableCell>
+                  <Text.H4 noWrap>
+                    {JSON.stringify(template.configuration)}
+                  </Text.H4>
+                </TableCell>
+                <TableCell>
+                  <Button
+                    variant='outline'
+                    onClick={() => destroy({ id: template.id })}
+                  >
+                    <Icon name='trash' />
+                  </Button>
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      }
+    />
+  )
+}
+
+function NewEvaluationTemplate() {
+  const { create } = useEvaluationTemplates()
+
+  const handleSubmit = (ev: FormEvent<HTMLFormElement>) => {
+    ev.preventDefault() // This line prevents the default form submission
+
+    const formData = new FormData(ev.currentTarget)
+    const name = formData.get('name') as string
+    const description = formData.get('description') as string
+    const prompt = formData.get('prompt') as string
```

Comment:
Why all this? Why not use `useLatitudeAction`

Reply:
idk i just moved this page component

Reply:
@geclos ☝️ This doesn't look like our standard way of doing mutations no?

Reply:
doesn't matter it's a backoffice that i don't expect to grow

Reply:
Agree to disagree then 🤝

Reply:
🍵 🫖